pywbem_mock/_mainprovider.py

```diff
@@ -678,12 +667,12 @@ def get_qualifier_store(self, namespace):
 
     def validate_namespace(self, namespace):
         """
-        Validates that a namespace is defined in the repository.
+        Validates that a namespace is defined in the datastore.
 
           Parameters:
 
             namespace (:term:`string`):
-              The name of the CIM namespace in the mock repository. Must not be
+              The name of the CIM namespace in the datastore. Must not be
               `None`.
```

Comment:
Describe that leading and trailing slashes are ignored, and that the lookup is case insensitive.

For example, the corresponding description in `BaseRepository.validate_namespace()` is:
"The name is treated case insensitively and it must not be `None`. Any leading and trailing slash characters in the namespace string are ignored when accessing the repository."

Reply:
DONE